src/services/share-buttons.service.ts

```diff
@@ -1,9 +1,7 @@
 import { Injectable, EventEmitter } from '@angular/core';
 import { Http, Jsonp, Headers, RequestOptions } from '@angular/http';
 
-import { Observable } from 'rxjs/Observable';
-import 'rxjs/add/operator/catch';
-import 'rxjs/add/Observable/empty';
+import { Observable } from 'rxjs';
```

Comment:
@jcao02 oh if that's the case then great 👍, cause it used to load the whole library before